### ISSUE TITLE
Regarding to lk_config.weight, lk_config.stable and so on.

### ISSUE THREAD

**Issue by taey16** [facebookresearch/supervision-by-registration#33]
First of all, I appreciate your efforts for sharing this repo. It gives chances to facilitate my research works.  
I’m reading your paper, and have several questions.
- First, in the paper, joint weight (\gamma in Eq.(7)) was set to 0.5, however, 0.2 (or 0.1) was used for mixing weight in this repo. I’d like to request for your explanation.
- Second, in the paper, you said connection between self-training and SBR. However, because of your default configuration (i.e. lk_config.stable = 1), your implementation seems to not back-propagate in \delta_theta LK(F_{t-1},F_t, L_{t-1,i}) term. Why did you introduce the lk_config.stable??
- Third, in the paper, you make use of VGG16-Reg network as a baseline model. But, I think regression loss can be used in training CPM jointly without the last fully-connected layer to regressing Kx2 points due to your soft-argmax function. How about this idea?

Thanks in advance.

**Comment by D-X-Y** [comment]
Thanks for your question.

(1) The 0.5 in Eq. (7) is used for jointly training 300VW/300W/YouTube-Face. While 0.1 is used for demo video only. The parameters of SBR is a little bit sensitive to different videos.

(2) Yes, "lk_config.stable=1" indicates the non-backpropagation version of SBR. During training, it is more stable than "lk_config.stable=0" backpropagation version. Because "lk_config.stable=0" will occasionally cause the loss=nan problem, which requires to resume the checkpoint from the last saved good model. I need to emphasize that "lk_config.stable=0" is theoretically correct, but not stable in practice, and we thus set lk_config.stable = 1 by default.

(3) Yes, I agree with you. Actually, we did try this: add regression loss as an auxiliary information to jointly train detection loss and regression loss. This will improve the performance by about 5%.
